data/projects/vulture.md

```diff
@@ -42,16 +42,19 @@ References:
 
 - Vulture offers its functionality as a library
 - The VultureBear uses the new vulture library
-
-##### CODING PHASE 2
-
 - Confidence values are implemented for vulture results
 
-##### CODING PHASE 3
+##### CODING PHASE 2
 
+- Configuring vulture so as to subsume a whitelist file by default.
 - If a way was found to offer source ranges, the removal of dead code is
   implemented for the VultureBear using a confidence value as threshold.
 - If no way was found, additional detection cases for vulture are implemented.
```

Comment:
Oh! I had it in my mind while writing this, how can I forget such a thing! :cold_sweat: 
Thanks!